test/unit/error.test.ts

```diff
@@ -254,14 +314,14 @@ describe('MongoErrors', () => {
     };
 
     before(() => (test = new ReplSetFixture()));
-    afterEach(() => mock.cleanup());
+    afterEach(() => cleanup());
     beforeEach(() => test.setup());
 
     function makeAndConnectReplSet(cb) {
       let invoked = false;
       const replSet = new Topology(
         [test.primaryServer.hostAddress(), test.firstSecondaryServer.hostAddress()],
-        { replicaSet: 'rs' }
+        { replicaSet: 'rs' } as any
```

Comment:
`//@ts-expect-error` goes here too right?

Reply:
Actually, no because `rs` is the correct type.  The better option is to directly cast as `TopologyOptions`, which I fixed 😄